tests/auto/micromouse/navigator_test.h.autosave

```diff
@@ -0,0 +1,22 @@
+#ifndef NAVIGATOR_TEST_H
```

Comment:
Delete this entire file.
Kevin probably has the autosave version still that needs saved